gulp/gulp-tests.js

```diff
@@ -41,6 +40,15 @@ const testBin = (string, additionalEnvVariables = '') => {
   return `NODE_ENV=test ${additionalEnvVariables} ${string}`;
 };
 
+function runInChildProcess (command, options = {}, envVariables = '') {
+  return done => pipe(exec(testBin(command, envVariables), options, done));
```

Comment:
Passing done directly as callback instead of the System.exit() option gets rid of the "did you forget to call done somewhere" notification when you have failing tests.